<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>851</width>
     <height>361</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="noise_tab">
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <attribute name="title">
     <string>Noise Tool</string>
    </attribute>
    <widget class="QPushButton" name="data_extraction_tab_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Extraction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stack_table_tab_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stack Table</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plot_tab_btn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stack_widget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 211, 211);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="extract_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 116, 118);</string>
       </property>
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="range_low_edit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="range_high_edit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>500</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="interest_freq_edit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1000, 2250</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lower</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Higher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Interests</string>
       </property>
      </widget>
      <widget class="QLabel" name="page2_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Extraction Control</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="prediction_only_box">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Prediction Only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="page2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Stack Control</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="from_selection_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>From Selection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="execute_stack_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 116, 118);</string>
       </property>
       <property name="text">
        <string>Stack</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="page3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot Control</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="sid_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Sid</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="sid_id2_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Sid/Id^2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="by_site_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 116, 118);</string>
       </property>
       <property name="text">
        <string>Plot by site</string>
       </property>
      </widget>
      <widget class="QPushButton" name="med_only_btn">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 116, 118);</string>
       </property>
       <property name="text">
        <string>Plot median</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="svg_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Svg</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="sid_f_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Sid*f</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filter_threshold_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0.10</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="threshold_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_filtered_btn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 116, 118);</string>
       </property>
       <property name="text">
        <string>Save Filtered Result</string>
       </property>
      </widget>
      <widget class="QLabel" name="tolerance_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tolerance: (± order)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filter_tolerance_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1.00</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTextEdit" name="output_path_text">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Output_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Output:</string>
     </property>
    </widget>
    <widget class="QListView" name="input_files_listview">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>411</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="Input_label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Input:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_selection_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Output</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_selection_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Input</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="debug_mode_box">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;

</string>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="filter_outliers_box">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(202, 0, 202);

</string>
     </property>
     <property name="text">
      <string>Enable filter</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
